### DISCUSSION TITLE
Pano Realing

### DISCUSSION THREAD

**Discussion by callmehanu** [oae/gnome-shell-pano#170] [Ideas]
I love Pano but think with some design tweaks could be a more productive and beautiful Gnome App.
Here a fast prototipo I worked!

👉️  [Figma Prototype](https://www.figma.com/proto/hTFYs9qIwhfeDewIkKWshE/Pano-Realing?node-id=1%3A2&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=1%3A2)

For this prototype try to launch Pano pressing `q`

![image](https://user-images.githubusercontent.com/13230502/224607002-678bd4ea-64fd-4f27-aa82-f335b5491d70.png)

**Comment by callmehanu** [comment]
Also I've added an interaction for searching... 
Just pressing ` i ` (for this case but really will work just typing after lauching Pano), in this case a search input will appear

**Comment by callmehanu** [comment]
@oae did you saw this?

**Reply by DuttaAB-dev** [reply]
Yes, he did. Rest assured
